src/SchemaFields/index.js

```diff
@@ -258,10 +258,6 @@ export const SelectWidget = ({
 
   const [isMenuOpen, setMenuOpen] = useState(false);
 
-  const SelectContainer = (props) => <div ref={containerRef}>
```

Comment:
Do we still need the `containerRef` if this was deleted?

Reply:
I'm not sure 🤔 honestly I'm trying to keep the surface area of changes as small as possible